code/game/machinery/rechargestation.dm

```diff
@@ -27,12 +28,14 @@
 		/obj/item/weapon/stock_parts/manipulator = 2,
 		/obj/item/weapon/stock_parts/capacitor = 2,
 		/obj/item/weapon/cell/high,
+		/obj/item/weapon/reagent_containers/glass/beaker,
 		/obj/item/stack/cable_coil{amount = 5}
 	)
 
 /obj/machinery/recharge_station/Initialize()
 	. = ..()
 	update_icon()
+	beaker.reagents.add_reagent("coolant", 60)

```

Comment:
How does this not runtime? beaker is null initially. YOu need to add `beaker = new/beaker(src)`